art/attacks/evasion/over_the_air.py

```diff
@@ -0,0 +1,238 @@
+# MIT License
+#
+# Copyright (C) The Adversarial Robustness Toolbox (ART) Authors 2021
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software
+# and associated
+# documentation files (the "Software"), to deal in the Software without restriction, including
+# without limitation the
+# rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+# Software, and to permit
+# persons to whom the Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all copies or
+# substantial portions of the
+# Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+# BUT NOT LIMITED TO THE
+# WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO
+# EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN
+# ACTION OF CONTRACT,
+# TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+# DEALINGS IN THE
+# SOFTWARE.
+
+"""
+This attack is an implementation of the over the air flickering attack from
+https://arxiv.org/pdf/2002.05123.pdf.
+"""
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import logging
+from typing import Optional, TYPE_CHECKING
+
+import torch
+from torch import nn
+from scipy.special import expit
+from tqdm.auto import trange
+
+from art.config import ART_NUMPY_DTYPE
+from art.estimators.estimator import BaseEstimator
+from art.estimators.classification.classifier import ClassGradientsMixin
+from art.attacks.attack import EvasionAttack
+from art.utils import compute_success, is_probability
+
+if TYPE_CHECKING:
+    from art.utils import CLASSIFIER_CLASS_LOSS_GRADIENTS_TYPE
+logger = logging.getLogger(__name__)
+
+
+class OverTheAirPyTorch(EvasionAttack):
+    attack_params = EvasionAttack.attack_params + ["regularization_param", "beta_1", "beta_2" "m"]
+    _estimator_requirements = (BaseEstimator, ClassGradientsMixin)
+
+    def __init__(
+        self,
+        classifier: "CLASSIFIER_CLASS_LOSS_GRADIENTS_TYPE",
+        regularization_param: float,
+        beta_1: float,
+        beta_2: float,
+        m: float,
```

Comment:
Could you please select a variable name with 3 or more characters, e.g.:
```suggestion
        margin: float,
```